contributions/authority/auth0/pom.xml

```diff
@@ -0,0 +1,195 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Copyright 2020 Yahoo Inc.
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+-->
+<project
+  xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd"
+>
+<modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.yahoo.athenz</groupId>
+  <artifactId>athenz_auth_auth0</artifactId>
+  <packaging>jar</packaging>
+  <version>1.0.1</version>
+  <name>athenz_auth_auth0</name>
+  <description>Athenz Yahoo Server OAuth2 access token implementation for Auth0</description>
+
+  <properties>
+    <athenz.version>1.8.45</athenz.version>
```

Comment:
this is quite an old version of athenz. let's use the latest release.

Reply:
Thank you for reminding.
As it depends on the latest auth_core, I just update to `1.8.60` first.
bc180c1f